app/src/main/java/de/tum/in/tumcampusapp/utils/StringExt.kt

```diff
@@ -0,0 +1,9 @@
+package de.tum.`in`.tumcampusapp.utils
+
+fun String.compareTo(other: String?): Int {
+    return if (other != null) {
+        compareTo(other)
+    } else {
+        -1
+    }
```

Comment:
I think this is quite literally [`compareValues`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.comparisons/compare-values.html)